### ISSUE TITLE
update personas

### ISSUE THREAD

**Issue by osteoarchaeo** [TheBiggerGuy/com.chelsislotten.www#42]
Personas should be horizontally listed not vertically stacked.  Images should be converted to text to increase accessibility.

**Comment by TheBiggerGuy** [comment]
WIP on https://github.com/TheBiggerGuy/com.chelsislotten.www/tree/html-persona branch

**Comment by osteoarchaeo** [comment]
Done!
